### DISCUSSION TITLE
HomeKit Reakation auf Variable

### DISCUSSION THREAD

**Discussion by daniel94ever** [thkl/hap-homematic#430] [Q&A]
Hy,
haben jetzt auch diese wirklich klasse AddOn gefunden und eingesetzt. Danke.

Da ich noch neu bin.

Was ist der beste Weg auf eine Variable (Berechnung Sonnenuntergang, true/false) in der HomeKit zu reagieren.
Der Versuch über einen virtuellen Taster
![image](https://user-images.githubusercontent.com/48876670/116774594-effdcc00-aa5d-11eb-802e-622f8304230d.png)
und Variablen
![image](https://user-images.githubusercontent.com/48876670/116774622-1cb1e380-aa5e-11eb-8f62-c8542b5eced6.png)
funktionieren nicht.

Habt ihr ein Beispiel.

**Comment by ChrWi** [comment]
Hallo @daniel94ever, du brauchst einen Trigger: https://github.com/thkl/hap-homematic/wiki/variables#trigger
